app/preferences.go

```diff
@@ -109,6 +109,10 @@ func (p *preferences) loadFromFile(path string) (err error) {
 
 	p.InMemoryPreferences.WriteValues(func(values map[string]interface{}) {
 		err = decode.Decode(&values)
+		if err != nil {
+			return
```

Comment:
Should we not do a `fyne.LogError()` on the value?

Reply:
I don't think so, no - this is exiting the write before applying conversion with an error set to the return value on the parent function call. So the caller will get the error and should log it if required.

Reply:
Alright. Sounds good.